routes/clams.js

```diff
@@ -0,0 +1,136 @@
+const express = require('express');
+const { pool } = require('../server/db');
+const { keysToCamel } = require('../common/utils');
+
+const router = express.Router();
+
+// Get data for all clams
+router.get('/', async (req, res) => {
+  try {
+    const clams = await pool.query('SELECT * FROM clams;');
+    res.status(200).json(keysToCamel(clams));
+  } catch (err) {
+    res.status(400).send(err.message);
+  }
+});
+
+// Get data for specific clam based on clamId
+router.get('/:clamId', async (req, res) => {
+  try {
+    const clam = await pool.query('SELECT * FROM clams WHERE clamId = $(clamId);');
+    res.status(200).json(keysToCamel(clam.rows));
+  } catch (err) {
+    res.status(400).send(err.message);
+  }
+});
+
+// Get data for clams based on rakerId
+router.get('/rakers/:rakerId/clams', async (req, res) => {
+  try {
+    const clam = await pool.query('SELECT * FROM clams WHERE rakerId = $(rakerId)');
```

Comment:
an object should be passed in to the query with your `rakerId` you're planning to insert into the query (see other backends' queries for examples)

tldr just test this route after you fix it and add the clams table